src/backend/XML/Structured.pm

```diff
@@ -8,7 +8,6 @@ require Exporter;
 @EXPORT            = qw(XMLin XMLinfile XMLout);
 $VERSION           = '1.1';
 
-use XML::Parser;
```

Comment:
@mlschroe : Please review carefully, might break things, but AFAICS this should be done in _chooseparser